watertap/property_models/NaCl_T_dep_prop_pack.py

```diff
@@ -1295,7 +1295,9 @@ def _therm_cond_phase(self):
             doc="Thermal Conductivity",
         )
 
-        def rule_therm_cond_phase(b, p):
+        def rule_therm_cond_phase(
+            b, p
+        ):  # Regressed from Zaytsev & Aseev (1992), 0-155 C
```

Comment:
```suggestion
        # Regressed from Zaytsev & Aseev (1992), 0-155 C
        def rule_therm_cond_phase(b, p):  
```

You didn't create the original issue, but we should get in the habit of putting comments *above* the code they're documenting, not to the right of it.

Otherwise `black` will make multi-line code out of otherwise single lines like here. I would encourage this change in every comment this PR touches.

Reply:
Thank you for pointing this out to me. I believe everything should be reformatted now.